tensorflow_tts/configs/parallel_wavegan.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 TensorFlowTTS Team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""ParallelWaveGAN Config object."""
+
+
+class ParallelWaveGANGeneratorConfig(object):
```

Comment:
We can move to dataclasses for configs it's a lot easier to write and cleaner (also it's a more pythonic way of doing things from py >=3.7).

```python3
from dataclasses import dataclass


@dataclass
class ParallelWaveGANGeneratorConfig(object):
    """Initialize ParallelWaveGAN Generator Config."""
    out_channels = 1,
    kernel_size = 3,
    n_layers = 30,
    stacks = 3,
    residual_channels = 64,
    gate_channels = 128,
    skip_channels = 64,
    aux_channels = 80,
    aux_context_window = 2,
    dropout_rate = 0.0,
    use_bias = True,
    use_causal_conv = False,
    upsample_conditional_features = True,
    upsample_params = {"upsample_scales": [4, 4, 4, 4]},
    initializer_seed = 42

```